styles/config/vocabularies/Technical/accept.txt

```diff
@@ -66,6 +67,7 @@ data's
 Deepspeed
 DigiCert
 Dockerfile[s]?
+[Dd]ockerhub
```

Comment:
this is correctly "[Docker Hub](https://hub.docker.com/)", from their own site

Reply:
Thank you for the clarification.